CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.1.1 (2018-12-20)
+
+This release contains bugfixes.
+
+**Notable commits**
+
+[f99b540] Avoid loading twice in parallel workers and load only from $libdir
+[e310f7d] Fixes to Pg_upgrade
```

Comment:
I prefer to keep the commit messages exact and in the section above actually describe the changes. I used to do a list titled `**High-level changes**` with then a bulleted list of things fixed. Maybe we go back to that (not sure why we stopped). So instead of 'This release contains bugfixes':
```
**High-level changes**
* Fix issue when upgrading with pg_upgrade
* Fix a segfault that sometimes appeared in long COPYs
* Other bug and stability fixes
```